WooCommerce/Classes/Tools/Logging/DefaultCrashLogger.swift

```diff
@@ -0,0 +1,26 @@
+import AutomatticTracks
+import Sentry
+import Storage
+
+final class DefaultCrashLogger: CrashLogger {
```

Comment:
Or `CrashLoggerProtocol`? 😁

Reply:
`SentryCrashLogger` sounds the best to me since it converts the severity level to `SentrySeverity`! updated in https://github.com/woocommerce/woocommerce-ios/pull/2366/commits/149d64927cae9cf43e46a42183e2e44493b27a32